### PR TITLE
Update stable install script text to account for viz

### PR DESCRIPTION
**Pull request by alpeb**
Followup to #936 for the install script that installs linkerd stable

### CODE REVIEW

File: run.linkerd.io/public/install [hunk 1]
```diff
@@ -16,7 +16,12 @@ happyexit() {
   echo "  linkerd check --pre                     # validate that Linkerd can be installed"
   echo "  linkerd install | kubectl apply -f -    # install the control plane into the 'linkerd' namespace"
   echo "  linkerd check                           # validate everything worked!"
-  echo "  linkerd dashboard                       # launch the dashboard"
+  echo ""
+  echo "You can also obtain observability features by installing the viz extension:"
```
Comment (wmorgan) [inline run.linkerd.io/public/install:20]: How about "Want the on-cluster metrics pipeline and dashboard? Run:"

**Review by wmorgan — APPROVED** [review]
Don't need to bikeshed this but it would be nice if all this output fit into 80 cols without wrapping. I think this could be done easily by trimming some whitespace and removing the namespace from the comments.

**Comment by alpeb** [comment]
@wmorgan I've shortened the comments. Please give it another look.

**Review by wmorgan — APPROVED** [review]
That works for me! Thank you.

**Pull request merged**